packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -159,4 +159,25 @@ api.grantMutation(role, 'updateExample');
 
 // For custom types and granular design
 api.grant(role, appsync.IamResource.ofType('Mutation', 'updateExample'), 'appsync:GraphQL');
-```
\ No newline at end of file
+```
+
+### s3-assets
+
+`aws-appsync` supports using s3 location for schema definition.
```

Comment:
clarify: 

does this mean it's supporting an S3 location to be referenced and used?
i.e. user already has an existing schema in S3 and they want to point to it

alternatively, does it mean users can author their schema definition locally and 
the cdk will upload to s3 and instrument it as an asset?

I think they're 2 different use cases, but I did get guidance from @rix0rrr that
we should address the former and ask the question as to whether the latter is 
really needed. Related resource where we had that discussion was for a Step Functions
state machine definition, which also supports an S3 location for defining a state machine.